app/interface/clients/__init__.py

```
@@ -0,0 +1,5 @@
+from .kutis_client import KutisClient
+from .lms_client import LmsClient
+from .barun_client import BarunClient
+
+__all__ = ["KutisClient", "LmsClient", "BarunClient"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)